src/SDDPoptimize.jl

```diff
@@ -169,14 +164,14 @@ Return:
 Float64 (estimation of the upper bound)
 
 """
-function estimate_upper_bound(model, param, V, problems, n_simulation=1000)
+function estimate_upper_bound(model::SPModel, param::SDDPparameters, V::Vector{PolyhedralFunction}, problem::Vector{JuMP.Model}, n_simulation=1000::Int)
 
     aleas = simulate_scenarios(model.noises, n_simulation)
 
     costs, stockTrajectories, _ = forward_simulations(model,
                                                         param,
                                                         V,
-                                                        problems,
+                                                        problem,
                                                         aleas)
 
     return upper_bound(costs), costs
```

Comment:
It looks like it return both the upper bound and the sequence of costs whereas the doc only state that it return the upper bound.